src/testdir/test_vim9_class.vim

```diff
@@ -74,7 +74,7 @@ def Test_class_basic()
       this
     endclass
   END
-  v9.CheckSourceFailure(lines, 'E1317: Invalid object variable declaration: this', 3)
+  v9.CheckSourceFailure(lines, 'E1318: Not a valid command in a class: this', 3)
```

Comment:
Can you add a test for using the prefix "variable" instead of "var" for the declaring an object variable?